### DISCUSSION TITLE
Video no Clipchamp

### DISCUSSION THREAD

**Discussion by jmtannus** [dcTeam23/course#7] [General]
Bom dia pessoal!

Mestre @AbraaoAlves, o clipchamp não aceita importar o arquivo com extensão .html nem .pdf

Quando clico no arquivo (com essas extensões) e arrasto para o local indicado, o Clipchamp mostra a mensagem abaixo:

"Não é possível importar algumas mídias,
O Clipchamp aceita apenas arquivos de imagem, áudio e vídeo."

Como o senhor faz?
E como acessar os comentários criados para a apresentação

**Comment by jmtannus** [comment]
Pra quem está com dúvidas - como eu - vou mostrar o passo a passo que estou tentando fazer. 
Pode não dar certo, mas vamos que vamos! 😄 

PASSO 1:
Vou começar a gravar a tela do desktop com a "ferramente de captura" que existe no Windows 11. 
Vou deixar o vídeo explicativo - bem didático, por sinal - do Adrenaline:
Como gravar a tela no Windows 11 sem programas! : https://youtu.be/h-xXX7HH9-s

depois volto contando se deu certo. 👍

**Reply by jmtannus** [reply]
ah, esqueci de dizer que tem que usar a ferramenta de captura de audio, simultaneamente. 🎙️ 
Após a gravação, vou recorrer ao Clipchamp, que nunca usei, mas vou aprender também...
Quase uma YouTuber! kkkkkk 
Um abraço e boa sorte a todos!

**Comment by AbraaoAlves** [comment]
> Bom dia pessoal!
> 
> Mestre @AbraaoAlves, o clipchamp não aceita importar o arquivo com extensão .html nem .pdf
> 
> Quando clico no arquivo (com essas extensões) e arrasto para o local indicado, o Clipchamp mostra a mensagem abaixo:
> 
> "Não é possível importar algumas mídias, O Clipchamp aceita apenas arquivos de imagem, áudio e vídeo."
> 
> Como o senhor faz? E como acessar os comentários criados para a apresentação

Você não precisa importar arquivos html, pdf ou qualquer texto. 

Para ter a experiência de gravar somente a tela da apresentação enquanto você pode ver a area dos comentários de apresentação, você pode seguir a seguinte opções:

  <img align="right" style="width:30%; height:auto;" src="https://github.com/dcTeam23/course/assets/608731/ddf923df-ea1d-417c-9882-005c5305acef"  width="300" height="225" />



1. Seja rico e tenha 2 monitores. Um para rodar a apresentação (onde o clipchamp vai capturar a tela) e o outro com os comentarios.

2. A outra opção é não gravar toda a tela, mas apenas [focar o clipchamp na janela que você deseja gravar](https://clipchamp.com/pt-br/blog/how-to-screen-record-audio/#passo-2-grave-seu-v%C3%ADdeo-da-tela), podendo abrir a tela do apresentador e mesmo assim não ter suas notas expostas na apresentação. Veja um tutorial completo nesse [link](https://clipchamp.com/pt-br/blog/how-to-screen-record-audio/#passo-2-grave-seu-v%C3%ADdeo-da-tela)

**Reply by jmtannus** [reply]
Consegui, muito obrigada!
Gravei o áudio e o video separado e editei no clipchamp. 
A gente salva como link e envia no discord, pra ti?
